aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -327,6 +339,15 @@ module aptos_framework::delegation_pool {
         commission_pending_inactive: u64,
     }
 
+    #[event]
+    struct DistributeCommissionEventV2 has drop, store {
```

Comment:
```suggestion
    struct DistributeCommission has drop, store {
```
Let's abandon `EventV2` suffix. v2 is more like an internal codename

Reply:
Renamed. Thanks!